kbase/narrative
# DATAUP-551 duplicate outputs

# Description of PR purpose/changes

Add errors to the bulk import config when duplicate outputs appear, and prevent the cell from running.
![Screen Shot 2021-08-12 at 2 53 24 PM](https://user-images.githubusercontent.com/2152243/129252894-c272db0b-4925-4dfa-b126-ecb9813c3983.png)

Note that this does not yet address the case where there's a duplicate output name across multiple file type tabs.

Also, tests missing right now. Working on it...
ETA: tests finished. Review away!

# Jira Ticket / Issue #
https://kbase-jira.atlassian.net/browse/DATAUP-551

# Testing Instructions
* Details for how to test the PR:
* Open a bulk import cell
* Add the same name to one or more output object names
* It should show an error under each line that's a duplicate.

# Dev Checklist:
